core/services/ocr2/plugins/directrequestocr/reporting.go

```diff
@@ -34,6 +36,43 @@ type functionsReporting struct {
 
 var _ types.ReportingPlugin = &functionsReporting{}
 
+var (
+	promReportingPlugins = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: "functions_reporting_plugin_instances",
```

Comment:
Maybe call it "reporting_plugin_restarts"? "instances" suggests that there can be more than one at the same time